index.js

```diff
@@ -405,14 +405,15 @@ class Tag extends React.PureComponent<TagProps> {
           this.props.tagContainerStyle,
         ]}
       >
-        <Text style={[
+        {React.isValidElement(this.props.label) ? this.props.label :
```

Comment:
I find that using ternary syntax inline in JSX feels pretty messy. Can you extract this condition to the start of the `render` function?